youtube.go

```diff
@@ -212,21 +212,22 @@ func (y *Youtube) parseVideoInfo() error {
 		return errors.New("no stream map found in the server's answer")
 	}
 
-	// Get video title and author.
-	title, author := getVideoTitleAuthor(answer)
-
 	var prData PlayerResponseData
 	if err := json.Unmarshal([]byte(streamMap[0]), &prData); err != nil {
 		fmt.Println(err)
 		panic("Player response json data has changed.")
 	}
 
-	// Get video download link
-	if prData.PlayabilityStatus.Status == "UNPLAYABLE" {
-		//Cannot playback on embedded video screen, could not download.
+	// Check if video is downloadable
+	if prData.PlayabilityStatus.Status == "UNPLAYABLE" || prData.PlayabilityStatus.Status == "LOGIN_REQUIRED" {
+		// Cannot playback on embedded video screen, could not download.
 		return errors.New(fmt.Sprint("Cannot playback and download, reason:", prData.PlayabilityStatus.Reason))
```

Comment:
By "LOGIN_REQUIRED" , the field "Reason" is empty.
please use custom message or Status instead

![image](https://user-images.githubusercontent.com/7224544/85636397-bbbcc080-b680-11ea-9a12-c84088e92a5a.png)

Reply:
good point! but sometimes I can see the field reason is not empty.
it could be `這是私人影片`. I could change it to show status only or show them both. which one do you prefer?

Reply:
in this case , my suggestion is to check if reason is empty string, then show status

Reply:
I just modify it accordingly  :)